rclcpp/include/rclcpp/topic_statistics/subscription_topic_statistics.hpp

```diff
@@ -127,7 +127,7 @@ class SubscriptionTopicStatistics
   /**
    * This method acquires a lock to prevent race conditions to collectors list.
    */
-  virtual void publish_message()
+  virtual void publish_and_reset_message()
```

Comment:
Nitpick: should be `publish_message_and_reset()`.

Reply:
Thanks.
I'd appreciate it if you could point out even the smallest details.